litgpt/__main__.py

```diff
@@ -0,0 +1,137 @@
+# Copyright Lightning AI. Licensed under the Apache License 2.0, see LICENSE file.
+
+from typing import TYPE_CHECKING, Any
+
+import torch
+
+if TYPE_CHECKING:
+    from jsonargparse import ArgumentParser
+
+
+def _new_parser(**kwargs: Any) -> "ArgumentParser":
+    from jsonargparse import ArgumentParser, ActionConfigFile
+
+    kwargs.setdefault("default_env", True)
```

Comment:
Can we disable this? It results in very large help pages because for every argument, it prints the ENV:

```
<function setup at 0x29a054360>:
  ARG:   --precision PRECISION
  ENV:   LITGPT_FINETUNE__LORA__PRECISION
                        (type: Optional[str], default: null)
  ARG:   --quantize QUANTIZE
  ENV:   LITGPT_FINETUNE__LORA__QUANTIZE
                        (type: Optional[Literal['nf4', 'nf4-dq', 'fp4', 'fp4-dq', 'int8-training']], default: null)
  ARG:   --devices DEVICES
  ENV:   LITGPT_FINETUNE__LORA__DEVICES
...
```

Reply:
Don't you think it could be useful to modify argument through the environment? For instance with studios?

Reply:
I don't see a use case right now. We will already generate very large help pages due to the scrips and so I think we should minimize additional info that is not applicable to the majority of users.